test/test_basic_rng/test_sample_indices.cc

```diff
@@ -88,13 +89,13 @@ class TestSampleIndices : public ::testing::Test
 
         RNGState state(seed);
         std::vector<int64_t> samples(num_samples, -1);
-        RandBLAS::util::sample_indices_iid_uniform(N, samples.data(), num_samples, state);
+        RandBLAS::util::sample_indices_iid_uniform(N, num_samples, samples.data(), state);
```

Comment:
This change should be accompanied by a change to the definition of ``RandBLAS::util::sample_indices_iid_uniform``.